project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.3.2" )
 addSbtPlugin("io.kevinlee" % "sbt-docusaur" % "0.9.0")
 addSbtPlugin("com.github.sbt" % "sbt-unidoc" % "0.5.0")
+addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.7.0")
```

Comment:
I don't think having the comment header with the license is necessary. 
@charleso What do you think?

Reply:
It isn't strictly necessary according to the f.a.q.  It's a practice in used to following because of some of my client work. munit uses Apache 2.0 as well, so the projects are compatible.

https://infra.apache.org/apply-license.html#copy-per-file

Reply:
From https://www.apache.org/legal/src-headers.html#purpose
> This document describes how Apache committers and PMC members should handle source file licensing and copyright notices.
It does not apply to developers outside the ASF who are applying the Apache License to their work. The [appendix to the Apache License](https://www.apache.org/licenses/LICENSE-2.0.html#apply) describes how others can apply the license to their work.

OK, so according to the page you shared, the practice from the page is for Apache Software Foundation (ASF), and based on what the [appendix to the Apache License](https://www.apache.org/licenses/LICENSE-2.0.html#apply), mentioned above, says having a `LICENSE` file is good enough I think.
> Include a copy of the Apache License, typically in a file called LICENSE, in your work, and consider also including a NOTICE file that references the License.

Reply:
👍🏻 Removing

Reply:
Headers removed with https://github.com/hedgehogqa/scala-hedgehog/pull/229/commits/4dbc441e2ae3f93e498f1d04949b70c1d3990db6